### ISSUE TITLE
Safari: focus styles for pagination links are not visible

### ISSUE THREAD

**Issue by greatislander** [pressbooks/pressbooks-network-catalog#57]
In Safari 15.6.1, the pagination links have focus styles that aren't visible:

<img width="226" alt="Screen Shot 2022-09-23 at 3 29 29 PM" src="https://user-images.githubusercontent.com/605361/192033689-6ceb9aec-e0f6-4c26-a7df-329d3ce40650.png">

In the above screenshot, the `previous` link is focused but has lost its normal outline. In my experience, Safari has some rendering issues related to customize `outline` properties. Using `box-shadow` might be more reliable across different browsers.
